### PR TITLE
fix(argo-cd): Align changelog structure to show changelogs on Artifact Hub

### PR DESCRIPTION
**Pull request by yu-croco**
relates to https://github.com/argoproj/argo-helm/issues/1808 .

Let's try new structure.... 🙏 
If this works well, I will update other projects' changelog structure and [here](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).

---

Signed-off-by: yu-croco <yu.croco@gmail.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

**Review by pdrastil — APPROVED** [review]

File: charts/argo-cd/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,5 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Changed]: Update Argo CD to v2.5.9"
+    - kind: fixed
+      description: Align changelog structure to show changelogs on Artifact Hub
```
Comment (yu-croco) [inline charts/argo-cd/Chart.yaml:27]: Let's try this structure.
ref: https://github.com/artifacthub/hub/blob/master/charts/artifact-hub/Chart.yaml#L82-L83

image

**Review by mbevc1 — APPROVED** [review]

**Comment by jmeridth** [comment]
LGTM.  We'll need to update [CONTRIBUTING.md](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md?plain=1#L83-L98) in this PR or a follow-up one.  Your call.

**Review by jmeridth — APPROVED** [review]
LGTM 👍

**Comment by pdrastil** [comment]
@jmeridth We will update - this is just a test to see if it resolves.

**Comment by yu-croco** [comment]
> We'll need to update [CONTRIBUTING.md](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md?plain=1#L83-L98) in this PR or a follow-up one. Your call.

Yes. If this works well, I will update other projects' changelog structure and CONTRIBUTING.md. 🙋

**Comment by jmeridth** [comment]
@yu-croco @pdrastil It worked.  Nice.

Screenshot 2023-01-29 at 9 52 26 AM

https://artifacthub.io/packages/helm/argo/argo-cd/5.19.12?modal=changelog&version=5.19.12

**Pull request merged**